docs/benchmarks.md

```diff
@@ -29,6 +29,27 @@ benchmark.
 | TPC-xBB   | com.nvidia.spark.rapids.tests.tpcxbb | TpcxbbLikeSpark, TpcxbbLikeBench |
 | TPC-H     | com.nvidia.spark.rapids.tests.tpch   | TpchLikeSpark, TpchLikeBench     |
 
+### Generating TPC-DS data in parallel
+
+The `dsdgen` tool provided by the [TPC-DS Data Generator](https://github.com/databricks/tpcds-kit) 
+is single-threaded and can take a long time to generate large scale factor data sets. The tool does
+provide support for generating one partition at a time, allowing the data generation to be run
+in parallel, but the generated files cannot be directly consumed by Apache Spark because they are
+not stored in a compatible directory structure. The 
```

Comment:
Curious why we wouldn't be using something like `TPCDSTables` from https://github.com/databricks/spark-sql-perf which is normally how `dsdgen` is scaled on a Spark cluster.

Reply:
Interesting. I didn't know about that. I will try it out.